src/core/featuremodel.h

```diff
@@ -29,8 +29,7 @@
 #include <qgsrelationmanager.h>
 
 #include <memory>
-
-class VertexModel;
+#include <vertexmodel.h>
```

Comment:
This include should be moved up with other QField source headers and use "double quotes" instead of <>, that'll help code readability by clearly stating that this is our header.